addons/html_editor/static/src/main/movenode_plugin.js

```diff
@@ -384,11 +384,13 @@ export class MoveNodePlugin extends Plugin {
                 p.append(br);
                 previousParent.append(p);
             }
-            const selectionPosition = endPos(movableElement);
-            this.shared.setSelection({
-                anchorNode: selectionPosition[0],
-                anchorOffset: selectionPosition[1],
-            });
+            if (movableElement.isContentEditable) {
```

Comment:
remove `movableElement.isContentEditable` as the problem should be handled in the selection plugin